src/pyFAI/detectors/sensors.py

```diff
@@ -156,7 +156,7 @@ def absorbance(self, energy:float, length: float, unit:str="m") -> float:
 ALL_MATERIALS["CdTe"] = CdTe_MATERIAL = SensorMaterial("CdTe", density=5.85)
 ALL_MATERIALS["GaAs"] = GaAs_MATERIAL = SensorMaterial("GaAs", density=5.3176)
 ALL_MATERIALS["Gd2O2S"] = Gd2O2S_MATERIAL = SensorMaterial("Gd2O2S", density=7.32)
-
+ALL_MATERIALS["BaFBr0.85I0.15"] = BaFBr085I015_MATERIAL =SensorMaterial("BaFBr0.85I0.15", density=3.3) 
 
```

Comment:
According to https://theses.hal.science/tel-04521313v1/file/SONG_PANYI_2023.pdf, p94, the density is 3.18g/cm^3, at least that is my understanding.
I would like to suggest to add as comment : `#https://theses.hal.science/tel-04521313v1/file/SONG_PANYI_2023.pdf, p94` as source.
I will change in my branch the key towards `BaFBr0.85I0.15`